README.md

```diff
@@ -45,7 +45,7 @@ Add the following to your `composer.json` file.
 ```json
 {
     "require": {
-        "sendgrid/smtpapi": "~0.6@dev"
+        "sendgrid/smtpapi": "^0.6.1"
```

Comment:
Need to add this to UpdateSendGridRubySMTPAPIFiles: https://code.hq.twilio.com/twilio/librarian/pull/178/files#diff-0b1d22901bca0c380969a6ee6b62cdd0 (sub of version number)

Reply:
That's ruby. PHP has it already.